docs/deployment/docker.md

```diff
@@ -39,7 +39,7 @@ package with `yarn build:backend`.
 
 In a CI workflow it might look something like this, from the root:
 
-```bash
+```sh
```

Comment:
Did this need to be changed now that bash is among the additional languages?

Reply:
Honestly, changed it before as everything else was `sh`. Happy to change it back

Reply:
Fixed 👍